packages/bits/demo/src/components/demo/tabgroup/tab-heading-group-with-router/tab-heading-group-with-router.example.component.html

```diff
@@ -1,10 +1,16 @@
 <div>
     <nui-tab-heading-group>
-        <nui-tab-heading *ngFor="let tab of tabsetContent" [tabId]="tab.id" routerLink="/tabgroup/{{tab.id}}" [active]="currentTabRoute === tab.id">
+        <nui-tab-heading
+            *ngFor="let tab of tabsetContent"
+            [tabId]="tab.id"
+            routerLink="/tabgroup/{{tab.id}}"
+            tabindex="-1"
+            [active]="currentTabRoute === tab.id"
+        >
```

Comment:
also by convention, Nova puts the closing bracket on the same line as the last attribute

Reply:
fixed, thanks